db/migrate/20200814124000_add_expiry_date_to_user_roles.rb

```diff
@@ -0,0 +1,5 @@
+class AddExpiryDateToUserRoles < ActiveRecord::Migration
+  def change
+    add_column :user_roles, :expiry_date, :date
```

Comment:
Probably best to use `:datetime` and compare to `Time.now` to be safe.

I can imagine a scenario where someone volunteers for one day, and we want to give permissions until the end of the work day (6pm) or something like that.

Not entirely sure it'll be needed, but better safe than sorry : )

Reply:
yes agree, let's go for `datetime` so we can specify end of work day